lib/std/http/Server.zig

```diff
@@ -424,7 +508,7 @@ pub const Response = struct {
 
         try w.writeAll("\r\n");
 
-        try buffered.flush();
+        try res.connection.flush();
```

Comment:
can afford to get rid of this one - there is a res.connection.flush() call in the res.finish() fn, so removing this one would reduce an extra write syscall per request.

up to you ...

Reply:
You're only required to call `Response.finish()` if you are also sending data, which isn't required.